platform/src/main/java/org/hillview/utils/TestTables.java

```diff
@@ -54,7 +54,7 @@ public static Table testTable() {
      * @return A small table with some repeated content.
      */
     public static Table testRepTable() {
-        ColumnDescription c0 = new ColumnDescription("Name", ContentsKind.Category, false);
+        ColumnDescription c0 = new ColumnDescription("Name", ContentsKind.String, false);
```

Comment:
Maybe you can avoid modifying this table, it is used in many places.
I guess no one depends on it being categorical.
Maybe you can change the column from StringArray to CategoryArray instead.

Reply:
Sure, I'll make a new test table with a String array.
> Maybe you can change the column from StringArray to CategoryArray instead.
I don't follow. The column (in the testRepTable) used to be a String before I changed it in my branch. Do you want me to change it to a categorical column? That's what I did!?

Reply:
The kind was Category but the implementation was StringColumn.
Use your new column instead, and don't change the kind.

Reply:
Ok, so it stays like this:
```
ColumnDescription c0 = new ColumnDescription("Name", ContentsKind.String, false);
```
That way the implementation stays StringColumn, as it was before the new Category implementation. I'll also change the `testTable()` (without the `Rep`) column in the same class.

(I see I switched the kinds around in my previous comment.)

Reply:
No keep c0 unchanged, but make sac be instead a CategoricalColumn

Reply:
Oh, right. Now I understand. Sorry for the confusion and my slow morning brain. ;-)